### ISSUE TITLE
Support different question sizes in dashboard subscription email/slack

### ISSUE THREAD

**Issue by Tony-metabase** [metabase/metabase#33635]
**Is your feature request related to a problem? Please describe.**
Let's for example take this dashboard in which i have a different size for the piechart:

<img width="1512" alt="image" src="https://github.com/metabase/metabase/assets/110378427/3eb63c83-3936-4d28-a9ec-6d9afd826303">

On the other hand I get the same size in the email:

<img width="359" alt="image" src="https://github.com/metabase/metabase/assets/110378427/cb278173-e407-479e-927b-664fb5c37384">

**Describe the solution you'd like**
Have different card sizes based on the card sizes in the dashboard

**Describe alternatives you've considered**
None are available

**Comment by brunobergher** [comment]
That's kind of by design, since the subscriptions go to a different medium, namely email. I think that pie chart is _way_ too big anyway, but custom sizes don't make a ton of sense for me in that context, and even less so on Slack.

Is there a motivation I'm missing here?

**Comment by Tony-metabase** [comment]
I agree but there is a specific customer that wants to be able to control the size of charts inside the email. If this feature request is something we will never consider I will simply close and revert back to the customer. 

Think of it this way if you want to put more emphases on a scatter plot which is 10x5 and you have a pie chart which is 3x3 you want the email to match the same context not both having the same size. That's the customer request.

**Comment by brunobergher** [comment]
Yeah. My take is that the right solution for that is to make the notification channels plug in, and having an "advanced email" one where people ca simply write their email templates, with HTML and something like {mustache}. We'd pass in dashcard info and they can customize it. I can totally see us doing that, it would be quite useful in embedding/whitelabeling contexts and handle all of the types of "remove links", "remove author", "remove filters" types of asks we get.

Let's leave this open, even if we don't tackle it for a little while.

**Comment by Tony-metabase** [comment]
That would be very cool actually. That is actually asked by a number of people. Not directly like that but is something that people ask a way to customize an email ... So definitely a plus for that!

Thanks for the feedback :)
